include/libpmemobj++/experimental/radix_tree.hpp

```diff
@@ -115,9 +115,9 @@ class radix_tree {
 	template <class InputIt>
 	radix_tree(InputIt first, InputIt last);
 
-	// radix_tree(const radix_tree& m);
-	// radix_tree(radix_tree&& m);
-	// radix_tree(initializer_list<value_type> il);
+	radix_tree(const radix_tree &m);
+	radix_tree(radix_tree &&m);
+	radix_tree(std::initializer_list<value_type> il);
 
```

Comment:
I just realized there should also be a constuctor which takes a pair of pointers https://en.cppreference.com/w/cpp/container/map/map (no 2, but without comparator/allocator)